### PR TITLE
pwn: add page

### PR DESCRIPTION
**Pull request by sufyandaredevil**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).

### CODE REVIEW

File: pages/linux/pwn.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# pwn
+
+> Exploit Development Library designed for rapid prototyping.
+> More information: <https://docs.pwntools.com/en/stable/commandline.html>.
+
+- Convert given assembly code to `bytes`:
+
+`pwn asm {{'xor edi, edi'}}`
```
Comment (gutjuri) [inline pages/linux/pwn.md:8]:
```suggestion
`pwn asm "{{xor edi, edi}}"`
```

File: pages/linux/pwn.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+- Create a cyclic pattern of `100` characters:
```
Comment (gutjuri) [inline pages/linux/pwn.md:10]:
```suggestion
- Create a cyclic pattern of 100 characters:
```

File: pages/linux/pwn.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+`pwn cyclic {{100}}`
+
+- Print `bytes` for the given `hex-encoded` data:
```
Comment (gutjuri) [inline pages/linux/pwn.md:14]:
```suggestion
- Encode the given data into the hexadecimal system:
```

File: pages/linux/pwn.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+
+`pwn hex {{deafbeef}}`
+
+- Print `hex-encoded` data for the given `bytes`:
```
Comment (gutjuri) [inline pages/linux/pwn.md:18]:
```suggestion
- Decode the given data from hexadecimal:
```

File: pages/linux/pwn.md [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+
+`pwn unhex {{6c4f7645}}`
+
+- Print `x64 Linux Shellcode` in bytes that runs a `shell`:
```
Comment (gutjuri) [inline pages/linux/pwn.md:22]:
```suggestion
- Print a x64 Linux shellcode for running a shell:
```

File: pages/linux/pwn.md [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+
+`pwn shellcraft {{amd64.linux.sh}}`
+
+- Check binary security settings for the given `ELF file`:
```
Comment (gutjuri) [inline pages/linux/pwn.md:26]:
```suggestion
- Check the binary security settings for the given ELF file:
```

File: pages/linux/pwn.md [hunk 7]
```diff
@@ -0,0 +1,36 @@
...
+
+`pwn checksec {{program}}`
```
Comment (gutjuri) [inline pages/linux/pwn.md:28]:
```suggestion
`pwn checksec {{path/to/file}}`
```

File: pages/linux/pwn.md [hunk 8]
```diff
@@ -0,0 +1,36 @@
...
+
+- Check for pwntools updates:
```
Comment (gutjuri) [inline pages/linux/pwn.md:30]:
```suggestion
- Check for Pwntools updates:
```

File: pages/linux/pwn.md [hunk 9]
```diff
@@ -0,0 +1,36 @@
...
+
+`pwn update`
+
+- Print Pwntools help menu:
```
Comment (gutjuri) [inline pages/linux/pwn.md:34]:
```suggestion
- Display version:
```

**Review by sufyandaredevil — COMMENTED** [review]
Suggestions applied 👍

**Review by gutjuri — APPROVED** [review]
Thanks

**Review by sufyandaredevil — COMMENTED** [review]
👍

**Review by sufyandaredevil — COMMENTED** [review]
Changes applied

**Review by kbdharun — APPROVED** [review]
Thanks for your contribution, LGTM after the above suggestion.

File: pages/linux/pwn.md [hunk 10]
```diff
@@ -0,0 +1,36 @@
...
+> Exploit Development Library designed for rapid prototyping.
+> More information: <https://docs.pwntools.com/en/stable/commandline.html>.
+
+- Convert  the given assembly code to `bytes`:
+
+`pwn asm "{{xor edi, edi}}"`
+
+- Create a cyclic pattern of 100 characters:
+
+`pwn cyclic {{100}}`
```
[the remark this answers is not in the document]
Comment (gutjuri) [reply pages/linux/pwn.md:12]: Should stay as-is in my opinion. The "100" can be changed by the user without really changing the meaning of the command, so it is a placeholder. We could change the "100" to "value" and also edit the description.

**Comment by kbdharun** [comment]
@sufyandaredevil Merging the branch with the main isn't required for most cases as we Squash & Merge PRs (you would only need to do it if the same file is modified somewhere else you need to see if the changes would conflict).

File: pages/linux/pwn.md (hunk 10 above)
Comment (kbdharun) [inline pages/linux/pwn.md:12]:
```suggestion
- Create a cyclic pattern of the specific number of characters:

`pwn cyclic {{value}}`
```

Comment (sufyandaredevil) [reply pages/linux/pwn.md:12]: I guess having {{value}} sounds more generic IMO since only numbers could be mentioned.

Comment (fazlearefin) [inline pages/linux/pwn.md:12]:
```suggestion
- Create a cyclic pattern of the specific number of characters:

`pwn cyclic {{number}}`
```

How about this?

**Review by sufyandaredevil — COMMENTED** [review]
Changes applied

**Comment by fazlearefin** [comment]
> Changes applied

Looks like the `pwn cyclic` command is still unresolved.

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

**Pull request merged**